### ISSUE TITLE
False Positive on SA1008 for Attributes on Array of ValueTuples

### ISSUE THREAD

**Issue by magicmonty** [DotNetAnalyzers/StyleCopAnalyzers#3117]
Code sample:

```csharp
private void UpdateSomething([NotNull] (string key, string value)[] keyValuePairs) 
{
}
```

SA1008 is thrown for the space between the tuple and the Attribute

**Comment by sharwell** [comment]
Do you know what version you are using? I see a bunch of tests for these cases added when #2472 was fixed.

**Comment by magicmonty** [comment]
I use currently 1.2.0-beta.164

**Comment by RobbeSchepens** [comment]
Take note of `public Point(int x, int y) => (X, Y) = (x, y);` which was giving a false positive on SA1008 but was solved in one of the latest betas. Just adding for info as it might collide with each other.
